_sql/sql_puzzle.sql

```diff
@@ -0,0 +1,13 @@
+-- Write your PostgreSQL query statement below
+WITH CTE_emp AS  
+( 
+SELECT d.name AS Department, e.name AS Employee,e.salary AS Salary ,
+DENSE_RANK() OVER (PARTITION BY d.name ORDER BY e.salary DESC )AS rn
+FROM Employee AS e
```

Comment:
I think we can remove the aliases or make them more descriptive. Usually, when SQL queries grow larger, it becomes harder to read with single-character aliases. Let's use more descriptive ones.